README.md

```diff
@@ -166,29 +166,14 @@ $ ./target/debug/ledger --pem tmp.pem balance
 ## `many` CLI
 Below are some examples of how to use the `many` CLI.
```

Comment:
Your PR is removing the examples. Why?

Reply:
@fmorency Appears to be a mistake . . . reverting . . .